src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -255,7 +255,7 @@ protected string DumpLabels()
         }
 #endif
 
-        private void EnterRegionIfNeeded(BoundNode node)
+        protected void EnterRegionIfNeeded(BoundNode node)
```

Comment:
If we remove the calls, then the accessibility changes in this file can be reverted

Reply:
Done